integration-tests/resteasy-reactive-kotlin/prod-mode/pom.xml

```diff
@@ -45,7 +45,7 @@
         <dependency>
             <groupId>org.jetbrains.kotlinx</groupId>
             <artifactId>kotlinx-coroutines-core-jvm</artifactId>
-            <version>1.4.2</version>
+            <version>${kotlinx-coroutines-core-jvm.version}</version>
```

Comment:
I think this is now in the BOM, no?

Reply:
There's no Kotlin coroutines version property available from this project.

Reply:
I see:

```xml
            <dependency>
                <groupId>org.jetbrains.kotlinx</groupId>
                <artifactId>kotlinx-coroutines-core-jvm</artifactId>
                <version>${kotlin.coroutine.version}</version>
            </dependency>
```

in the BOM, so the version can be dropped here

Reply:
Great, thanks!